h2o-algos/src/main/java/hex/glm/ComputationState.java

```diff
@@ -569,8 +633,7 @@ public int compare(Integer o1, Integer o2) {
   }
 
   protected boolean checkKKTsMultinomial(){
-    if(_activeData._activeCols == null) return true;
-    throw H2O.unimpl();
+    return true;  // todo: implement this properly
```

Comment:
is this ok?

Reply:
This is supposed to check if KKT conditions are satisfied.  However, all it is doing is checking if activeCols are null or not which basically does nothing here.  While this function is not actually implemented at all, so it is okay here.  I have written a reminder about actually checking it.  The real check has something to do with checking the actual coefficient value changes.